vite.config.js

```diff
@@ -1,9 +1,13 @@
 import { resolve } from 'path'
 import { defineConfig } from 'vitest/config'
+import vue from '@vitejs/plugin-vue'
 import dts from 'vite-plugin-dts'
 
 export default defineConfig({
   test: {
+    typecheck: {
+      checker: 'vue-tsc',
+    },
```

Comment:
Interesting, haven't seen this before.

Reply:
typechecking was failing on import of .vue file, this solved it